lib/checks/canFindDenominatorInNumerator.js

```diff
@@ -19,17 +20,42 @@ function canFindDenominatorInNumerator(node) {
   if (Node.Type.isParenthesis(denominator)) {
     denominator = denominator.content;
   }
+  if (!(numerator.op === '+' || numerator.op === '-' ||
+        denominator.op === '+' || numerator.op === '-')) {
+    return false;
+  }
+  if (denominator.op !== '+') {
+    return false;
+  }
+
+  let numeratorFirstTerm;
+  if (numerator.op === '+') {
+    numeratorFirstTerm = new Node.PolynomialTerm(numerator.args[0]);
+  }
+  else if (numerator.op === '*') {
+    numeratorFirstTerm = new Node.PolynomialTerm(numerator);
+  }
+
+  let denominatorFirstTerm;
+  if (denominator.op === '+') {
+    denominatorFirstTerm = new Node.PolynomialTerm(denominator.args[0]);
+  }
+  else if (denominator.op === '*') {
+    denominatorFirstTerm = new Node.PolynomialTerm(denominator);
+  }
 
-  if (numerator.op !== '+') {
+  if (!(numeratorFirstTerm)) {
     return false;
   }
-  if (!('args' in denominator)) {
+  if (!(denominatorFirstTerm)) {
+    return false;
+  }
+
+  if (!(numeratorFirstTerm.getSymbolName() === 'x' && denominatorFirstTerm.getSymbolName() === 'x')) {
```

Comment:
wait I'm not really getting the use of getSymbolName() actually, what other values can the symbol even take, it seems like it always returns 'x'. Is this just for when there are multiple variables? Cause that seems way out of the scope for now anyways. Would it be better to check that denominatorFirstTerm.getExponentNode() === numeratorFirstTerm.getExponentNode() or check if they equal undefined?